src/assets/browser/app/src/composit/stage/mod.rs

```diff
@@ -0,0 +1,5 @@
+mod stage;
```

Comment:
(Whole file) separate "stage" position management into own module. Only lightly coupled with rest of package and easier to factor when methods are private to a small module.